project/plugins.sbt

```diff
@@ -3,7 +3,7 @@ libraryDependencies += Defaults.sbtPluginExtra(
   (pluginCrossBuild / sbtBinaryVersion).value,
   (pluginCrossBuild / scalaBinaryVersion).value)
 
-addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.7.0")
+addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.8.0")
```

Comment:
Do we have something corresponding to scalafmtCheck enabled in CI for the java formatter? Otherwise there is a likely that PRs are merged with wrong format and then someone else has to include unrelated changes later. If the tool doesn't have support for it we can add a git diff check. We had that previously and we can dig it up in history, if it has been removed.

Reply:
We have `formatOnCompile` enabled for javafmt (see `JavaFormatter.scala`).